lib/bindings/langs/swift/Sources/BreezSDKLiquid/Task/NwcEvent.swift

```diff
@@ -0,0 +1,37 @@
+import UserNotifications
+import Foundation
+
+struct NwcEventNotification: Codable {
+    let eventId: String
+}
+
+class NwcEventTask: TaskProtocol {
+    fileprivate let TAG = "NwcEventTask"
+
+    internal var payload: String
+    internal var contentHandler: ((UNNotificationContent) -> Void)?
+    internal var bestAttemptContent: UNMutableNotificationContent?
+    internal var logger: ServiceLogger
+
+    init(payload: String, logger: ServiceLogger, contentHandler: ((UNNotificationContent) -> Void)? = nil, bestAttemptContent: UNMutableNotificationContent? = nil) {
+        self.payload = payload
+        self.contentHandler = contentHandler
+        self.bestAttemptContent = bestAttemptContent
+        self.logger = logger
+    }
+
+    func start(liquidSDK: BindingLiquidSdk, plugins: PluginServices) throws {
+        var request: NwcEventNotification? = nil
+        do {
+            request = try JSONDecoder().decode(NwcEventNotification.self, from: self.payload.data(using: .utf8)!)
+            try plugins.nwc!.handleEvent(eventId: request!.eventId)
+        } catch let e {
+            self.logger.log(tag: TAG, line: "failed to decode payload: \(e)", level: "ERROR")
+            throw e
+        }
+    }
+
+    func onEvent(e: SdkEvent) {}
+    func onShutdown() {}
```

Comment:
Same here, on iOS we definitely need to respond to the notification request with displaying notification content